portafly/.env.development

```diff
@@ -0,0 +1 @@
+PORT=3003
```

Comment:
Why do we need this? we environment loader in the project.

Reply:
I think it's meant to be ignored, same as `.env`

Reply:
no guys, please, read the commit message: https://github.com/3scale/porta/pull/1626/commits/aafae3618b5fe45fe4b5011fa4183e8c9d93ef4b

Reply:
Perhaps a comment here that explains it? So that nobody ignores/removes it from the project?

Reply:
Or perhaps we start reading commit messages as they were meant/created for ;)

Reply:
I read the commit message and the [section in the React Docs about this](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env), and I don't like this either. Can we consider making this an "example" file, like https://github.com/3scale/porta/tree/master/config/examples?

Reply:
Or perhaps just in the README should do it. In Porta too we have plenty of env vars and configs that, when non-optional like APICAST_REGISTRY_URL, we always include in the command to run the server.

Reply:
I like the idea of the example file as we have plenty in porta, plus a reinforcement in portafly readme (not porta one)

Reply:
Maybe better in the Readme only, since this is about env variables, people might want to use .envs or in the command line, or in their .zshrc/.bashrc/etc